tracer/test/Datadog.Trace.Tests/LambdaRequestBuilderTests.cs

```diff
@@ -13,14 +14,17 @@
 
 namespace Datadog.Trace.Tests
 {
+    extern alias DatadogTraceManual;
+
     [Collection(nameof(WebRequestCollection))]
     public class LambdaRequestBuilderTests
     {
         [Fact]
         public async Task TestGetEndInvocationRequestWithError()
         {
             await using var tracer = TracerHelper.CreateWithFakeAgent();
-            var scope = LambdaCommon.CreatePlaceholderScope(tracer, traceId: null, samplingPriority: null);
+            var headers = new WebHeaderCollection();
```

Comment:
Will need to wrap all of these (or move the wrapping _inside_ `CreatePlaceholderScope` if you prefer)